types/packages/caver-rpc/src/klay.d.ts

```diff
@@ -170,8 +170,8 @@ export class Klay {
 
     call(callObject: CallObject, callback?: (error: Error, result: string) => void): Promise<string>
     call(callObject: CallObject, blockNumber: BlockNumber, callback?: (error: Error, result: string) => void): Promise<string>
-    estimateGas(callObject: CallObject, callback?: (error: Error, result: string) => void): Promise<string>
-    estimateGas(callObject: CallObject, blockNumber: BlockNumber, callback?: (error: Error, result: string) => void): Promise<string>
+    estimateGas(callObject: CallObject, callback?: (error: Error, result: string) => void): Promise<number>
```

Comment:
I don't think `caver.rpc.klay.estimateGas` will return number.
You can check `caver-rpc/src/klay.js` file, there is no output formatter.
But for `caver.klay.estimateGas` have output formatter which converts to number type from hex string. (You can check in `rpc.json` file)
Can you revert this? You can just fix return type for `types/packages/caver-rtm/src/index.d.ts` file, which is `caver.klay.estimateGas` type.

Reply:
@jimni1222 Sorry, I thought it will return `number`.
fixed it at 59bb723. please check.